worlds/saving_princess/docs/setup_en.md

```diff
@@ -0,0 +1,120 @@
+3. If you don't have `original_data.win`, copy `data.win` and rename its copy to `original_data.win`
+   * By keeping an unmodified copy of `data.win`, you will have an easier time updating in the future
+4. Apply the `saving_princess_basepatch.bsdiff4` patch using your patching software
+5. To launch the game, run `Saving Princess v0_8.exe`
+
+### Manual Linux Installation
+
+The game does run mostly well through Wine, so it is possible to play on Linux, although there are some minor sprite displacement and sound issues from time to time.
+
+You can follow the instructions for Windows with very few changes:
+
+* Using the p7zip package to decompress the file.
+```
+7z e 'Saving Princess.exe'
+```
+* And the bsdiff package for patching.
+```
+bspatch original_data.win data.win saving_princess_basepatch.bsdiff4
+```
+
+## Configuring your YAML file
+
+### What is a YAML file and why do I need one?
+
+See the guide on setting up a basic YAML at the Archipelago setup
+guide: [Basic Multiworld Setup Guide](/tutorial/Archipelago/setup/en).
+
+### Where do I get a YAML file?
+
+You can customize your options by visiting the [Saving Princess Player Options Page](/games/Saving Princess/player-options).
+
+### Verifying your YAML file
+
+If you would like to validate your config file to make sure it works, you may do so on the YAML Validator page. YAML
+validator page: [YAML Validation page](/check).
+
+## Joining a MultiWorld Game
+
+### Set up the connection details
+
+After launching the game, enter the Archipelago options menu through the in-game button with the Archipelago icon.
+From here, enter the different menus and type in the **server:port**, **slot**, as well as the **password**, if one is required.
+
+This configuration persists through launches and even updates.
+
+### Connect to the Archipelago Server
+
+To start a connection attempt, simply press on "CONNECT!".
+
+Once connected, the button will become one of either "NEW GAME" or "CONTINUE".
+The game automatically keeps a save file for each seed and slot combination, so you do not need to manually move or delete save files.
+
+All that's left is pressing on the button again to start playing. If you are waiting for a countdown, press "NEW GAME" when the countdown finishes.
+
+## Gameplay Questions
+
+### Do I need to save the game before I stop playing?
+
+It is safe to close the game at any point while playing, your progress will be kept.
+
+### What happens if I lose connection?
+
+If a disconnection occurs, you will see the HUD connection indicator go grey.
+From here, the game will automatically try to reconnect.
+You can tell it succeeded if the indicator regains its color.
+
+If the game is unable to reconnect, save and restart.
+
+Although you can keep playing while disconnected, you won't get any items until you reconnect, not even items found in your own game.
+Once reconnected, however, all of your progress should sync up.
```

Comment:
By "should sync up" I was trying to say that gm-apclientpp would take care of it.
I guess I should just say "will sync up"? Because if we assume the mod, the client and the protocol all work, then it will sync up, and if doesn't I don't think there's anything the user could really do to help it anyway?

The only way I've managed to make it not sync properly in my testing has been by wiping all of the local server's data and then starting it back up, but if it comes to that I'm not sure there's much a user can do about it.

Reply:
Changing to "will" makes sense to me then